src/main/config/thurloe.conf.ctmpl

```diff
@@ -0,0 +1,22 @@
+{{with $environment := env "ENVIRONMENT"}}{{$keyname := printf "secret/dsde/%s/thurloe/thurloe.conf" $environment}}{{with vault $keyname}}
+
+include "application.conf"
+
+database {
+  config = {{.Data.db_config}}
+
+  main {
+    google-dev {
```

Comment:
Actually, if this template is used in other environments, "google-dev" may not be an appropriate name for this DB config. Maybe rename to "cloudsql" (and correct the reference to it above)?